unified-codes/workspace.json

```diff
@@ -16,8 +16,7 @@
             "main": "apps/data-service/src/app/main.ts",
             "tsConfig": "apps/data-service/tsconfig.app.json",
             "assets": ["apps/data-service/src/assets"],
-            "generatePackageJson": true,
-            "externalDependencies": ["fastify"]
```

Comment:
Hmmm, I don't know the details because I didn't do the original hosting setup but from (https://github.com/openmsupply/unified-codes/pull/199) it seems like this the externalDependency was needed for hosting to work? @mark-prins would you perhaps remember something from 6 months ago? 🤣

Reply:
Hmm, it should work now that the `package.json` is being generated (at least when it's dockerised). I'm not having any problems running locally or dockerised, but let me know if you are having any!

Reply:
Hm okies, no issues locally for me, guess we can just keep in mind for next deploy 🤞